app/models/task.py

```diff
@@ -1,5 +1,27 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    is_complete = db.Column(db.Boolean, default=False)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    goal = db.relationship('Goal', back_populates='tasks')
+    
+    def to_dict(self):
+        task_as_dict = {}
+        task_as_dict["id"] = self.task_id
+        task_as_dict["title"] = self.title
+        task_as_dict["description"] = self.description
+        # task_as_dict["completed_at"] = self.completed_at
+        task_as_dict["is_complete"] = self.is_complete
+        task_as_dict["goal_id"] = self.goal_id
```

Comment:
Ideally, we'd want to avoid setting this attribute if the attribute's value is `None`. This will likely cause tests to fail.